benchmarks/src/main/scala/zio/BubbleSortBenchmarks.scala

```diff
@@ -1,12 +1,12 @@
 package zio
 
-import java.util.concurrent.TimeUnit
+import IOBenchmarks.unsafeRun
```

Comment:
I think there may be something off with the logic here. It looks like this is getting moved up because it starts with `IO` but it is `IO` versus `io` and not followed by a `.`.

Reply:
IO vs io ?

Reply:
you mean IO vs org? I am confused. This does not seem to be case sensitive.

Reply:
Wouldn't we want the order to be `java`, `scala`, and then everything else? Why are `IOBenchmarks` being ordered first?

Reply:
Ahhhh I still have not correct the configuration. I was experimenting to get some feedback. Let me reformat
:DDDD